project3/styles.css

```diff
@@ -0,0 +1,424 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+:root {
+    --color-blue: #163346;
+    --color-gray: #d1d5de;
+    --color-offwhite: #fff;
+    --color-lightgray: #cececd;
+    --color-pureblack: #000000;
+    --color-offblack: #1e2019;
+    --color-lime: #72f79f;
+
+    --font-teko: 'Teko', sans-serif;
+    --font-bench: 'BenchNine', sans-serif;
+
+}
+
+body {
+    background-color: var(--color-blue);
+    color: var(--color-offwhite);
+    font-family: var(--font-bench);
+    font-size: 22px;
+    margin: 0;
+}
+
+
+header {
+    background-color: var(--color-gray);
+    display: block;
+}
+
+
+main {
+    background-color: var(--color-offblack);
+    margin: 20px;
+    padding: 15px;
+    position: relative;
+    top: -100px;
+    left: 0;
+}
+
+
+div {
+    margin-bottom: 10px;
+}
+
+
+
+h1,
+p {
+    text-align: center;
+    color: var(--color-pureblack);
+}
+
+
+
+h1 {
+    font-family: var(--font-teko);
+    font-size: 48px;
+    line-height: 1;
+    margin-bottom: 20px;
+    text-transform: uppercase;
+
+}
+
+h2 {
+    border-bottom: 5px solid var(--color-blue);
+    color: var(--color-offwhite);
+    display: inline-block;
+    font-family: var(--font-teko);
+    font-size: 40px;
+    font-weight: 600;
+    line-height: 1;
+    margin-bottom: 30px;
+    margin-top: 80px;
+    text-transform: uppercase
+}
+
+
+header p {
+    color: var(--color-blue);
+    margin: auto;
+    padding-bottom: 140px;
+
+}
+
+
+
+fieldset p {
+    color: var(--color-lightgray);
+    margin: 0;
+    text-align: left;
+
+}
+
+
+fieldset {
+    border: 0;
+    margin: 0;
+    padding: 0;
+
+}
+
+
+input,
+select,
+textarea {
+    font-family: inherit;
+    font-size: inherit;
+
+}
+
+legend {
+    margin-bottom: 0;
+}
+
+
+label {
+    display: block;
+
+}
+
+label[for=strength],
+label[for=speed],
+label[for=intelligience], {
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+select {
+    cursor: pointer;
+}
+
+textarea {
+    width: 100%;
+}
+
+input {
+    margin-bottom: 15px;
+    width: 100%;
+
+}
+
+
+select {
+    background: gray;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 40px;
+    text-align: center;
+    width: 100%;
+
+}
+
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid var(--color-lightgray);
+    cursor: pointer;
+    height: 20px;
+    width: 20px;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: aqua;
+    display: inline-block;
+}
+
+
+input[type="radio"] {
+    border-radius: 50%
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    border: 2px solid var(--color-lime);
```

Comment:
For `hover` and `focus` effects, it's better to put a transparent border of the proper width on the element and then only change the `border-color` when the state changes. This way, the input doesn't appear to resize or move when the border is added.